src/bmeg/util/cli.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
These types of headers aren't needed.

"Python supports writing source code in UTF-8 by default"
https://docs.python.org/3.3/howto/unicode.html#unicode-literals-in-python-source-code

And the shebang line is only needed in scripts which a user would execute directly.